ledger/catchpointfilewriter_test.go

```diff
@@ -705,16 +711,8 @@ func testNewLedgerFromCatchpoint(t *testing.T, catchpointWriterReadAccess tracke
 	err = accessor.BuildMerkleTrie(context.Background(), nil)
 	require.NoError(t, err)
 
-	resetAccountDBToV6(t, l)
-
-	err = l.trackerDBs.Transaction(func(ctx context.Context, tx trackerdb.TransactionScope) error {
-		cw, err := tx.MakeCatchpointWriter()
-		if err != nil {
-			return err
-		}
-
-		return cw.ApplyCatchpointStagingBalances(ctx, 0, 0)
-	})
+	// Initializes DB, runs migrations, runs ApplyCatchpointStagingBalances
+	err = accessor.(*catchpointCatchupAccessorImpl).finishBalances(context.Background())
```

Comment:
I had to go read the function, but basically you are removing a shortcut we are doing in favor of the identical cleanup/reset code used live.